plugins/repositories/core/pom.xml

```diff
@@ -32,36 +31,6 @@
   </properties>
 
   <dependencies>
-    <dependency>
-      <groupId>com.fasterxml.jackson.core</groupId>
-      <artifactId>jackson-core</artifactId>
-      <version>${fasterxml.jackson.version}</version>
-    </dependency>
-    <dependency>
-      <groupId>com.fasterxml.jackson.core</groupId>
-      <artifactId>jackson-annotations</artifactId>
-      <version>${fasterxml.jackson.version}</version>
-    </dependency>
-    <dependency>
-      <groupId>com.fasterxml.jackson.core</groupId>
-      <artifactId>jackson-databind</artifactId>
-      <version>${fasterxml.jackson.version}</version>
-    </dependency>
-    <dependency>
-      <groupId>com.fasterxml.jackson.jaxrs</groupId>
-      <artifactId>jackson-jaxrs-json-provider</artifactId>
```

Comment:
Repositories plugin has a json provider declared in its blueprint as such it seems this dependency shouldn't be removed. Please double check.
https://github.com/pentaho/pentaho-kettle/blob/master/plugins/repositories/core/src/main/resources-filtered/OSGI-INF/blueprint/beans.xml#L82